Extensions/go_to_dash.js

```diff
@@ -25,28 +25,29 @@ XKit.extensions.go_to_dash = new Object({
 		if ($("#xkit_gotodash").length > 0) {
 			return;
 		}
+		
+		XKit.tools.init_css("go_to_dash");
 
 		var post_id = XKit.iframe.single_post_id();
 		var next_post_id = parseInt(post_id) + 1;
 
-		// The CSS class 'embed-button' uses Tumblr's CSS to add padding
-		// to the button itself.
-		var go_back_html = '<a href="/dashboard/2/' + next_post_id + '" class="tx-button btn embed-button" target="_top" ' +
+		var go_back_html = '<a href="/dashboard/2/' + next_post_id + '" class="tx-icon-button" target="_top" ' +
 			'id="xkit_gotodash" title="View on dashboard"><span class="button-label">View</span></a>';
 
 		// Remove the text from the dashboard button because otherwise the iframe
 		// overflows
 		XKit.iframe.hide_button(XKit.iframe.dashboard_button());
 
 		var place = XKit.iframe.unfollow_button().length ?
-			XKit.iframe.unfollow_button() : XKit.iframe.delete_button();
+			XKit.iframe.dashboard_button() : XKit.iframe.delete_button();
```

Comment:
this is a behavioral change—now go-to-dash will show up on blogs that you're not following